### PR TITLE
Add quantile_at_value function

### PR DESCRIPTION
**Pull request by skrzypo987**
## Release notes

( ) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
(x) Release notes are required, with the following suggested text:

```markdown
# Section
* Add quantile_at_value function
```

### CODE REVIEW

**Review by Praveen2112 — COMMENTED** [review]
Minor nits

File: core/trino-main/src/test/java/io/trino/operator/aggregation/TestQuantileDigestFunctions.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        assertions = null;
+    }
+
+    @Test
+    public void testQuantileAtValueBigint()
+    {
+        QuantileDigest qdigest = new QuantileDigest(1);
+        addAll(qdigest, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
+        assertThat(assertions
+                .expression(format("quantile_at_value(CAST(a AS qdigest(bigint)), 20)"))
```
[the remark this answers is not in the document]
Comment (skrzypo987) [reply core/trino-main/src/test/java/io/trino/operator/aggregation/TestQuantileDigestFunctions.java]: Excelent idea. Added as another commit.

**Review by Praveen2112 — APPROVED** [review]

**Comment by skrzypo987** [comment]
Commits squashed.
I don't know if red CI is just flakiness. Should I rerun?

**Comment by Praveen2112** [comment]
Thank you for adding the new functions.

**Pull request merged**